appeals/functions/hr-data-import/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+	"IsEncrypted": false,
+	"Values": {
+		"FUNCTIONS_WORKER_RUNTIME": "node",
+		"AzureWebJobsStorage": "UseDevelopmentStorage=true",
+		"API_HOST": "http://localhost:3000"
```

Comment:
Should API_HOST contain the protocol as the POST request has it hardcoded to https?

Reply:
That's a local settings only, will be replaced by the deployment pipelines... A bit pointless as testing end-to-end not really an option, so the functions are just a msg forwarder to the API.